### PR TITLE
Docs: more removal of dbt for T&D

### PR DESCRIPTION
**Pull request by evantahler**

### CODE REVIEW

File: docs/understanding-airbyte/basic-normalization.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Basic Normalization
 
+:::danger
+
+Basic normalization is being removed in favor of [Typing and Deduping](/understanding-airbyte/typing-deduping), as part of [Destinations V2](/release_notes/upgrading_to_destinations_v2). This pages remains as a guide for legacy connectors.
```
Comment (edgao) [inline docs/understanding-airbyte/basic-normalization.md:5]: do we want to specify which connectors are already on dv2 and which are still upcoming?

**Review by edgao — COMMENTED** [review]
generally lgtm, one small comment on the warning wording. will leave it up to @Hesperide for final approval since it's a nontrivial docs change

File: docs/understanding-airbyte/basic-normalization.md (hunk 1 above)
Comment (evantahler) [reply docs/understanding-airbyte/basic-normalization.md:5]: If we do, it should be on one of the linked pages rather than this one

**Comment by evantahler** [comment]
@Hesperide ping!

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**